src/amber/server/settings.cr

```diff
@@ -1,26 +1,63 @@
 require "./cluster"
-require "./configuration"
 require "./ssl"
 
 module Amber
   class Settings
     include Amber::DSL::Server
-    property port_reuse : Bool
-    property log : ::Logger
-    property color : Bool = true
-    property process_count : Int32
-    property secret_key_base : String
-    property port : Int32
-    property name : String
-    property host : String
-    property ssl_key_file : String? = nil
-    property ssl_cert_file : String? = nil
-    property redis_url = ""
+    property log : ::Logger = Logger.new(STDOUT)
```

Comment:
I don't think we should have properties of settings that can't be set through environment.yms's.

Reply:
I agree with you. But for now I am going to leave as is for now until we can come up with a better soloution. WDYT?